subscription/scout-details.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Docker Scout subscriptions and features
+description: Learn about the Docker Scout subscriptions plans and features
+keywords: subscription, free, team, business, features
+---
+
+{% include scout-early-access.md %}
+
+This page provides an overview of the subscription plans for Docker Scout. To compare features available for each plan, see [Docker Scout Pricing](https://docker.com/products/docker-scout){: target="_blank" rel="noopener" class="_"}.
+
+## Docker Scout Free 
+
+Docker Scout Free is available for both personal accounts and organizations. This plan is ideal for individual developers getting started with software supply chain security.
+
+Docker Scout Free includes:
+
+- Unlimited local image analysis
+- Up to 3 Docker Scout-enabled repositories
+- Vulnerability analysis
+- Image comparison for vulnerabilities, size, and packages
+- Recommendations for refreshing or updating the base image
+- SDLC integration, including policy evaluation and workload integration
+- On-prem and cloud container registry integrations
+- Security posture reporting
```

Comment:
**suggestion:** Looks like the list of features has been trimmed.

<img width="1306" alt="Screenshot 2023-06-30 at 10 26 48 AM" src="https://github.com/docker/docs/assets/2487951/2d843d92-5922-4409-9ef3-968c390a2594">

@cdupuis could you please confirm?

Reply:
Confirmed on Slack to update docs to match the trimmed list: https://docker.slack.com/archives/C04C69EM70C/p1688152892696629?thread_ts=1688022705.075599&cid=C04C69EM70C